juspay/hyperswitch
# feat(openapi): add payment get to openapi

## Type of Change



## Description

This PR adds the payments get endpoint to the openapi. This PR also fixes some errors that were introduced in https://github.com/juspay/hyperswitch/pull/6393, which did not allow the v2 application to be run.


## Motivation and Context



## How did you test it?

- Run the v2 application
- Run mintlify and check for the new endpoint
<img width="1483" alt="Screenshot 2024-11-12 at 11 50 03 AM" src="https://github.com/user-attachments/assets/bed78286-4a58-4d36-9f0b-9e3e43cdd10b">


## Checklist
